cdap-ui/app/features/tracker/controllers/metadata-ctrl.js

```diff
@@ -331,12 +336,25 @@ class TrackerMetadataController {
 
   goToTag(event, tag) {
     event.stopPropagation();
-    this.$state.go('search.objectswithtags', {tag: tag});
+    this.$state.go('tracker.detail.result', {searchQuery: tag});
+  }
+
+  openTagInput(event) {
+    event.stopPropagation();
+    this.inputOpen = true;
+
+    this.eventFunction = () => {
+      this.escapeInput();
+    };
+    document.body.addEventListener('click', this.eventFunction, false);
```

Comment:
Not necessarily in this PR but we should do something similar to modal backdrop. Would be self contained. OK for now.